sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -137,7 +137,7 @@ def __init__(
                 "or value_body being set as the body of the AmqpAnnotatedMessage."
             )
 
-        self._body_type: AmqpMessageBodyType = None # type: ignore
+        self._body_type: AmqpMessageBodyType
```

Comment:
is there any chance that `self._body_type` wouldn't get assigned here? like if the following kwargs below weren't passed?

Reply:
the lines right above check that there is 1 of data/sequence/value_body in the kwargs